ext/coverage/coverage.c

```diff
@@ -75,6 +75,35 @@ rb_coverage_result(VALUE klass)
     return ncoverages;
 }
 
+static int
+coverage_merge_result_i(st_data_t key, st_data_t val, st_data_t h)
+{
+    VALUE path      = (VALUE)key;
+    VALUE coverage  = (VALUE)val;
+    VALUE coverages = (VALUE)h;
+    VALUE ary;
+
+    ary = rb_hash_lookup2(coverages, path, Qnil);
+
+    if (RTEST(ary)) {
+      rb_ary_replace(ary, coverage);
```

Comment:
If you use Emacs or Vim, https://bugs.ruby-lang.org/projects/ruby/wiki/DeveloperHowto#For-Emacs has how to specify.